### PR TITLE
Added productResourceFeedback

### PR DESCRIPTION
**Pull request by pliew**
Sales channel specific command.  ResourceFeedback is on the application level whereas this is on the product level

### CODE REVIEW

**Comment by lpinca** [comment]
Can you please change the name from `productResourceFeedback` to `resourceFeedback` as per https://help.shopify.com/en/api/reference/sales-channels/resourcefeedback?

**Comment by lpinca** [comment]
Perhaps something like this https://github.com/MONEI/Shopify-api-node#metafields using `resource_type` and `resource_id` can be used with the already existing `ResourceFeedback` to work on a specific resource?

**Comment by pliew** [comment]
> ~Can you please change the name from `productResourceFeedback` to `resourceFeedback` as per https://help.shopify.com/en/api/reference/sales-channels/resourcefeedback?~
> 
> Is there any official documentation for this? I can't find it.

https://help.shopify.com/en/api/reference/sales-channels/productresourcefeedback
Looks like they hide it but it's been available since 2019.  We've been discussing this with the internal team at Shopify and it's not a new feature

**Comment by pliew** [comment]
> Perhaps something like this https://github.com/MONEI/Shopify-api-node#metafields using `resource_type` and `resource_id` can be used with the already existing `ResourceFeedback` to work on a specific resource?

The reasoning I chose this method is that the resource id is based off the REST url and not based on a post/query parameter.

**Comment by lpinca** [comment]
As long as it is documented I'm fine with it. Give me some time to review. Thanks.

File: resources/product-resource-feedback.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+/**
+ * Creates a new product variant.
+ *
+ * @param {Number} productId Product ID
+ * @param {Object} params Product variant properties
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+ProductResourceFeedback.prototype.create = function(productId, params) {
```
Comment (lpinca) [inline resources/product-resource-feedback.js]: The default one from the base-child mixing should work unless am I missing something can you please remove?

File: resources/product-resource-feedback.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * @param {Shopify} shopify Reference to the Shopify instance
+ * @constructor
+ * @public
+ */
+function ProductResourceFeedback(shopify) {
+  this.shopify = shopify;
+
+  this.parentName = 'products';
+  this.name = 'resource_feedback';
+  this.key = 'resource_feedback';
```
Comment (lpinca) [inline resources/product-resource-feedback.js]:
```suggestion
  this.key = this.name = 'resource_feedback';
```

File: resources/product-resource-feedback.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ *
+ * @param {Shopify} shopify Reference to the Shopify instance
+ * @constructor
+ * @public
+ */
+function ProductResourceFeedback(shopify) {
+  this.shopify = shopify;
+
+  this.parentName = 'products';
+  this.name = 'resource_feedback';
```
Comment (lpinca) [inline resources/product-resource-feedback.js]:
```suggestion
```

**Comment by lpinca** [comment]
Can you please also add documentation for this on README.md?

**Pull request merged**